### PR TITLE
[DPEDE-776] Transfer list - Create HTML/CSS (Vue2)

### PR DESCRIPTION
**Pull request by pedrotaglialatela-lumen**

### CODE REVIEW

File: src/chi-vue/src/components/transfer-list/TransferListColumn.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      </p>
+      <div class="transfer-list-column-header__actions">
+        <chi-icon icon="atom" size="sm--3" />
+        <chi-icon icon="atom" size="sm--3" />
+      </div>
+    </div>
+    <div class="transfer-list-column__search">
+      <chi-search-input placeholder="Filter" />
+    </div>
+    <div class="transfer-list-column__content">
```
Comment (jllr) [inline src/chi-vue/src/components/transfer-list/TransferListColumn.vue:18]:
```suggestion
    <div class="chi-transfer-list__menu">
```

File: src/chi-vue/src/components/transfer-list/TransferListColumn.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <div v-for="(item, index) in list" :key="index" class="transfer-list-item">
```
Comment (jllr) [inline src/chi-vue/src/components/transfer-list/TransferListColumn.vue:19]:
```suggestion
      <div v-for="(item, index) in list" :key="index" class="chi-transfer-list__menu-item">
```

File: src/chi/components/transfer-list/transfer-list.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
+.transfer-list-column {
+  width: 200px;
```
Comment (jllr) [inline src/chi/components/transfer-list/transfer-list.scss:2]: Also use rems in height, width...

File: src/chi/components/transfer-list/transfer-list.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  flex-direction: column;
+  gap: 0.625rem;
+
+  &__content {
+    border: 0.0625rem solid $transfer-list-border-color;
+    border-radius: $transfer-list-border-radius;
+    position: relative;
+    overflow: auto;
+    height: 300px;
+  }
```
Comment (jllr) [inline src/chi/components/transfer-list/transfer-list.scss:13]:
```suggestion
    height: 300px;
    overflow: auto;
    position: relative;
  }
```

File: src/chi/components/transfer-list/transfer-list.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+.transfer-list-column-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  &__title {
+    display: flex;
+    align-items: center;
+    gap: 0.3125rem;
+    color: $transfer-list-title-text-color;
```
Comment (jllr) [inline src/chi/components/transfer-list/transfer-list.scss:25]:
```suggestion
    color: $transfer-list-title-color;
```

File: src/chi/components/transfer-list/transfer-list.scss [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    font-weight: 700;
```
Comment (jllr) [inline src/chi/components/transfer-list/transfer-list.scss:26]: There is a variable for font-weights too

File: src/chi/components/transfer-list/transfer-list.scss [hunk 7]
```diff
@@ -0,0 +1,51 @@
+.chi-transfer-list {
+  display: flex;
+  flex-direction: column;
+  gap: 0.625rem;
+  width: 12.5rem;
+
+  &__menu {
+    border: 0.0625rem solid $transfer-list-border-color;
+    border-radius: $transfer-list-border-radius;
```
Comment (jllr) [inline src/chi/components/transfer-list/transfer-list.scss:9]:
```suggestion
    border-radius: $transfer-list-menu-border-radius;
```

File: src/chi-vue/src/components/transfer-list/TransferListColumn.vue [hunk 8]
```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="chi-transfer-list">
```
Comment (jllr) [inline src/chi-vue/src/components/transfer-list/TransferListColumn.vue:2]:
```suggestion
  <div class="chi-transfer-list__column">
```

File: src/chi-vue/src/components/transfer-list/TransferListColumn.vue [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+    <div class="chi-transfer-list__header">
+      <p class="chi-transfer-list__title">
+        {{ title }}
+        <slot name="chi-transfer-list-title-tooltip"></slot>
+      </p>
+      <div class="chi-transfer-list__header-actions">
+        <slot name="chi-transfer-list-header-actions"></slot>
```
Comment (jllr) [inline src/chi-vue/src/components/transfer-list/TransferListColumn.vue:9]:
```suggestion
        <slot name="header-actions"></slot>
```

File: src/chi-vue/src/components/transfer-list/TransferListActions.vue [hunk 10]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="chi-transfer-list__actions">
+    <button class="chi-button -icon -flat" type="button">
+      <i class="chi-icon icon-chevron-right"></i>
+    </button>
+    <button class="chi-button -icon -flat" type="button">
+      <i class="chi-icon icon-chevron-left"></i>
+    </button>
```
Comment (jllr) [inline src/chi-vue/src/components/transfer-list/TransferListActions.vue:8]: @pedrotaglialatela-lumen could we change these buttons to web components? Same as we have in TransferListColumn.vue line 8 

```suggestion
    <button class="chi-button -icon -flat" type="button">
      <i class="chi-icon icon-chevron-right"></i>
    </button>
    <button class="chi-button -icon -flat" type="button">
      <i class="chi-icon icon-chevron-left"></i>
    </button>
```

Comment (pedrotaglialatela-lumen) [reply src/chi-vue/src/components/transfer-list/TransferListActions.vue:8]: @jllr sure!! Updating

**Review by jllr — APPROVED** [review]

**Pull request merged**